homeassistant/components/tomorrowio/weather.py

```diff
@@ -280,15 +330,20 @@ def _forecast(self, forecast_type: str) -> list[Forecast] | None:
                 self._forecast_dict(
                     forecast_dt,
                     use_datetime,
+                    cloud_coverage,
```

Comment:
Change the field order to match the order the fields were requested in __init__.py